### ISSUE TITLE
buildah from image ID not work

### ISSUE THREAD

**Issue by gouyang** [projectatomic/buildah#142]
```
[root@fedora buildah]# ./buildah from aa04e7c462d1f94ef011ddfc3d4e04149b1ff8af7da174b52ab38c700bfa46e4
ERRO[0000] error parsing reference to image "aa04e7c462d1f94ef011ddfc3d4e04149b1ff8af7da174b52ab38c700bfa46e4": invalid repository name (aa04e7c462d1f94ef011ddfc3d4e04149b1ff8af7da174b52ab38c700bfa46e4), cannot specify 64-byte hexadecimal strings 
[root@fedora buildah]# ./buildah from aa04e7c462d1
ERRO[0003] error pulling image "aa04e7c462d1": Error initializing image from source docker://aa04e7c462d1:latest: errors:
denied: requested access to the resource is denied
unauthorized: authentication required
```

**Comment by rhatdan** [comment]
@nalind Didn't this one get fixed?

**Comment by rhatdan** [comment]
I believe this is fixed in the current buildah 0.4 release.
